packages/frontend/app/dashboard/components/user-dashboard.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+Just for now before I can figure out how to programatically generate these
+dashboards based on the user type
+*/
+/* eslint-disable react/jsx-key */
```

Comment:
Same here

Reply:
This will be sorted out later once all the user data is set up properly